bfcfg

```diff
@@ -834,6 +837,8 @@ sys_cfg()
   sys_cfg_one_byte ${tmp_file} "ENABLE_FORCE_BOOT_RETRY" 53 "${SYS_ENABLE_FORCE_BOOT_RETRY}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_OEM_MFG_CONFIG" 54 "${SYS_ENABLE_OEM_MFG_CONFIG}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_I2C1" 55 "${SYS_DISABLE_I2C1}"
+  sys_cfg_one_byte ${tmp_file} "ENABLE_DDR_5600" 56 "${SYS_ENABLE_DDR_5600}"
```

Comment:
Not sure we would like to support this config from Linux -- We need to check with Shravan. Last time he said it is not needed.